docs/index.rst

```diff
@@ -3,7 +3,8 @@ Mila technical documentation
 
 .. First and only part to the documentation.
 
-Welcome to Mila's technical documentation. See contents below.
+Welcome to Mila's technical documentation. See contents below and have a look
+into the :ref:`short quick start <mila_code>`.
```

Comment:
NOTE: I don't know how the end of this sentence will display (I don't know the .rst format).
```suggestion
Welcome to Mila's technical documentation. If this is your first time here, we recommend you start by checking out the :ref:`short quick start guide <mila_code>`.
```